service/qos/service_level_controller.cc

```diff
@@ -634,17 +634,31 @@ future<scheduling_group> service_level_controller::auth_integration::get_user_sc
         auto sl_opt = co_await find_effective_service_level(*usr->name);
         auto& sl_name = (sl_opt && sl_opt->shares_name) ? *sl_opt->shares_name : default_service_level_name;
         co_return _sl_controller.get_scheduling_group(sl_name);
-    }
-    else {
+    } else {
```

Comment:
This is a stylistic cleanup unrelated to the fix. Consider doing it in a separate commit.